modules/mocked_authentication/spec/lib/mocked_credential/service_spec.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'mocked_credential/service'
+
+describe MockedAuthentication::MockedCredential::Service do
+  let(:code) { '6805c923-9f37-4b47-a5c9-214391ddffd5' }
+  let(:token) do
```

Comment:
Many of these `let` values aren't being used in tests, so they should be removed